### ISSUE TITLE
How much gain can Jigsaw bring?

### ISSUE THREAD

**Issue by lihao0374** [HobbitLong/PyContrast#7]
Good job! I was wondering how much gain can Jigsaw bring ?

**Comment by HobbitLong** [comment]
As in the paper, it would help improve from 63.6% to 65.9% with 100 epochs training with a linear projection head.

No, those hyper-parameters are NOT tuned in the `InfoMin` paper. For the Color Jittering, it was tuned in [SimCLR](https://arxiv.org/abs/2002.05709); for the crop_size of 0.2, it was used in [MoCo](https://arxiv.org/abs/1911.05722), where they say they followed InstDis and I don't know whether they have tuned it.
So in short, I just plot what is happening behind these parameter choices provided by previous papers.

I would close it for now, but please feel free to reopen it if you have further questions.

**Comment by lihao0374** [comment]
I ’m sorry for my reckless question. Is it possible for you to show the gain that Jigsaw can bring?

**Comment by lihao0374** [comment]
> As in the paper, it would help improve from 63.6% to 65.9% with 100 epochs training with linear projection head.
> 
> No, those hyper-parameters are NOT tuned in the `InfoMin` paper. For the Color Jittering, it was tuned in [SimCLR](https://arxiv.org/abs/2002.05709); for the crop_size of 0.2, it was used in [MoCo](https://arxiv.org/abs/1911.05722), where they say they followed InstDis, but I feel they might have tuned it.
> So in short, I just plot what is happening behind these parameter choices provided by previous papers.
> 
> I would close it for now, but please feel free to reopen it if you have further questions.

I ’m sorry for my reckless question. Is it possible for you to show the gain that Jigsaw can bring?

**Comment by lihao0374** [comment]
> As in the paper, it would help improve from 63.6% to 65.9% with 100 epochs training with linear projection head.
> 
> No, those hyper-parameters are NOT tuned in the `InfoMin` paper. For the Color Jittering, it was tuned in [SimCLR](https://arxiv.org/abs/2002.05709); for the crop_size of 0.2, it was used in [MoCo](https://arxiv.org/abs/1911.05722), where they say they followed InstDis, but I feel they might have tuned it.
> So in short, I just plot what is happening behind these parameter choices provided by previous papers.
> 
> I would close it for now, but please feel free to reopen it if you have further questions.

Thanks for your reply ~

**Comment by HobbitLong** [comment]
No worry at all :) It’s actually a reasonable question!

Jigsaw improves it from 63.6 to 65.9. But I haven’t ablated it in the full model

**Comment by lihao0374** [comment]
> No worry at all :) It’s actually a reasonable question!
> 
> Jigsaw improves it from 63.6 to 65.9. But I haven’t ablated it in the full model

Sorry to bother you again, could you tell me how to get the optimal parameters N=2 and M=10 in RA?

**Comment by HobbitLong** [comment]
Please refer to here: #3

**Comment by lihao0374** [comment]
> Please refer to here: #3

Many thanks ！
